plugin/compile-scss.js

```diff
@@ -126,13 +128,28 @@ class SassCompiler extends MultiFileCachingCompiler {
         }
       }
 
+      for (const possibleFile of possibleFiles) {
+        if (!possibleFile.startsWith('./') && !possibleFile.startsWith('{}') && !isAbsolute) {
+          possibleFiles.push('./' + possibleFile);
+          possibleFiles.push('{}/' + possibleFile);
+        }
+      }
+
+      for (const possibleFile of possibleFiles) {
+        if (possibleFile.startsWith('./') && !possibleFile.startsWith('{}') && !isAbsolute) {
+          possibleFiles.push('{}/' + possibleFile.substring('./'.length));
+        }
+      }
+
```

Comment:
This changes the `possibleFiles` array while looping over it. Avoid that and instead do something like this:

```javascript

if (!isAbsolute) {
  let additionalPossibleFiles = [];
  for (const possibleFile of possibleFiles) {
    if(...) {
      additionalPossibleFiles.push(...)
    }
  }
  possibleFiles = possibleFiles.concat(additionalPossibleFiles);
}
```

Also I'm wondering why it is necessary to add those variants. Is this something required by `dart-sass` or is it an additional feature you like to include in `meteor-scss` package?

Reply:
> This changes the possibleFiles
I'd gladly do it but I tried to go along the project code style. It's already done it a bunch of places. Should we go forward with immutability in only one place, leaving it mutable in older code? If not, shouldn't it be a matter of a separate pull request (probably, together with code formatting)?

Reply:
> Is this something required by dart-sass or is it an additional feature you like to include in meteor-scss package?

Rechecked with my project, doesn't seem to be needed indeed.